[ID] Add "Ranking Criteria Council" (#5403)
ppy/osu-wiki: wiki/Ranking_Criteria/Ranking_Criteria_Council/en.md

- ![][flag_CL] [ZiRoX](https://osu.ppy.sh/users/200768) (American mappers)
- ![][flag_US] [Zak](https://osu.ppy.sh/users/1375955) (American players)
- ![][flag_GB] [JBHyperion](https://osu.ppy.sh/users/4879508) (European mappers)
- *(vacant)* (European players)
- ![][flag_KR] [Spectator](https://osu.ppy.sh/users/702598) (Asian mappers)
- ![][flag_SG] [He Ang Erika](https://osu.ppy.sh/users/2451381) (Asian players)
- ![][flag_ES] [Deif](https://osu.ppy.sh/users/318565) (QAT)
- ![][flag_HU] [Kurokami](https://osu.ppy.sh/users/260933) (QAT)

### osu!mania members

- ![][flag_CA] [Fullerene-](https://osu.ppy.sh/users/2531335) (American mappers)
- ![][flag_US] [Halogen-](https://osu.ppy.sh/users/169992) (American players)
- ![][flag_PL] [-Kamikaze-](https://osu.ppy.sh/users/2124783) (European mappers)
- ![][flag_GB] [Hayabusa](https://osu.ppy.sh/users/3104108) (European players)
- ![][flag_US] (![][flag_KR]) [Jinjin](https://osu.ppy.sh/users/3360737) (Asian mappers)
- ![][flag_MY] [Cryolien](https://osu.ppy.sh/users/1626983) (Asian players)
- ![][flag_GB] [Starry-](https://osu.ppy.sh/users/2166199) (QAT)
- ![][flag_US] [Blocko](https://osu.ppy.sh/users/4075092) (QAT)

> The Ranking Criteria Council was ultimately unsuccessful. Only members involved in osu!catch promoted new proposals, while the other game modes did not make progress until the Ranking Criteira Council was disbanded and replaced with the United Beat-Knights of Ranking Criteria.

Reviewer: ```suggestion
The Ranking Criteria Council was ultimately unsuccessful. Only members involved in osu!catch promoted new proposals, while the other game modes did not make progress until the Ranking Criteria Council was disbanded and replaced with the United Beat-Knights of Ranking Criteria.
```

Reviewer 2: I think this one need to be reopened @Nivalyx . There is typo on the second sentence where `Criteira` should be `Criteria`

Author: oh i did not notice this earlier whoops

fixed!

## United Beat-Knights of Ranking Criteria

> The **United Beat-Knights of Ranking Criteria** (*UBKRC*) was a new group with similar goals to the Ranking Criteria Council, however it was handled with less strictness. The group chose an intentionally stupid name to appear more friendly than its predecessor. The group was managed by ![][flag_DE] [Okoratu](https://osu.ppy.sh/users/1623405) and ![][flag_US] [pishifat](https://osu.ppy.sh/users/3178418) until completion.

Reviewer: ```suggestion
The **United Beat-Knights of Ranking Criteria** (*UBKRC*) was a new group with similar goals to the Ranking Criteria Council, however it was handled with less strictness. The group chose an intentionally stupid name to appear more friendly than it's predecessor. The group was managed by ![][flag_DE] [Okoratu](https://osu.ppy.sh/users/1623405) and ![][flag_US] [pishifat](https://osu.ppy.sh/users/3178418) until completion.
```

> Within the UBKRC, teams of experts were chosen to handle each area of the general Ranking Criteria and game-mode specific sections. Users could join if they showed an active interest in any group.

Reviewer: ```suggestion
 Within the UBKRC, teams of experts were chosen to handle each area of the general Ranking Criteria and game-modes specific sections. Users could join if they showed an active interest in any groups.
```

> Between 2016 and 2018, the UBKRC reformed all sections of the Ranking Criteria to fit modern standards. Following its completion, the Ranking Criteria subforum was re-opened for community-submitted proposals.

Reviewer: ```suggestion
 Between 2016 and 2018, the UBKRC reformed all sections of the Ranking Criteria to fit modern standards. Following it's completion, the Ranking Criteria subforum was re-opened for community-submitted proposals.
```
